packages/core/src/blocks/ListItemBlockContent/CheckListItemBlockContent/CheckListItemBlockContent.ts

```diff
@@ -118,18 +118,7 @@ const checkListItemBlockContent = createStronglyTypedTiptapNode({
   parseHTML() {
     return [
       {
-        tag: "div[data-content-type=" + this.name + "]", // TODO: remove if we can't come up with test case that needs this
-        getAttrs: (element) => {
```

Comment:
why are these parts removed exactly? they're parsed somewhere else?

Reply:
Yeah idk when we added `getAttrs` here but it's not needed and causes bugs - we already parse props in `propsToAttributes`.